apps/challenges/models.py

```diff
@@ -54,3 +87,22 @@ class ChallengePhase(TimeStampedModel):
     class Meta:
         app_label = 'challenges'
         db_table = 'challenge_phase'
+
+    def __str__(self):
+        """Returns the name of Phase"""
+        return self.name
+
+    def get_start_date(self):
+        """Returns the start date of Phase"""
+        return self.start_date
+
+    def get_end_date(self):
+        """Returns the end date of Challenge"""
+        return self.end_date
+
+    @property
+    def is_active(self):
+        """Returns if the challenge is active or not"""
+        if self.end_date > timezone.now():
+            return True
+        return False
```

Comment:
No, we are not planning to have get method for each field. We will have methods for those fields which we need to use somewhere and some other methods that will devise some metric using the fields already present such as `is_active()`. 

Sure, I will add a test for that.